### PR TITLE
feat(Test): add sample for testing proto3 custom options

### PR DESCRIPTION
**Pull request by zakhenry**
This PR is more of a request for comment - ts-proto doesn't yet support proto3 [custom options](https://protobuf.dev/programming-guides/proto3/#customoptions) (they are defined in proto2 docs, but are actually well supported in proto3). I've added a sample proto of the kinds of fields that can be used, and want to start discussion on how this might be incoporated into the ts-proto api (behind which flags etc)

### CODE REVIEW

File: integration/extensions-proto3/parameters.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+outputExtensions=true,unknownFields=true,outputServices=generic-definitions,outputServices=default
```
Comment (zakhenry) [inline integration/extensions-proto3/parameters.txt:1]: I initially thought I could get this working with `outputExtensions=true`, but actually I suspect this needs to be an entirely new flag `outputCustomOptions=true` ?

**Comment by zakhenry** [comment]
wait this might just be https://github.com/stephenh/ts-proto/issues/570. I'll sanity check myself then close if it's covered by that flag

**Comment by zakhenry** [comment]
yea it is, whoops

**Comment by stephenh** [comment]
Np @zakhenry , good find that there is an existing issue for this -- re-reading that issue, sounds like there were some distractions/complications from the `Exact` type, but that otherwise the `outputSchema` afaiu includes options...

Granted, probably as part of its larger "schema for all your messages" output, but in theory it has code that reads the options, that could then be included in the grpc service definition output...

If you'd like to make a PR for #570 that just always outputs options into the grpc service output (if I'm following along), that'd be great! Thanks!

**Pull request closed without merging**